webapp/src/client/client.js

```diff
@@ -5,6 +5,8 @@ import AuthenticationContext from 'adal-angular';
 import {isDesktopApp} from '../utils/user_utils';
 import {Periods} from '../constants';
 import {id as pluginID} from '../manifest';
+import { getPluginServerRoute } from 'selectors';
```

Comment:
To stay consistent with the other imports

```suggestion
import {getPluginServerRoute} from 'selectors';
```

Reply:
@mickmister This is Done